### DISCUSSION TITLE
zwave-js not working after HA update 2022.5

### DISCUSSION THREAD

**Discussion by espenthingstad** [zwave-js/zwave-js#4721] [General]
Hi

I have tried to update HA to 2022.5, but when I do so, the zwave integration does not work. I see in the documentation that:
"If you use the zwave_js add-on, you need to have at least version 0.1.56."

I find in my instance that I have: 
Driver Version: 8.11.6
Server Version: 1.15.0

Do I need to update, and if so - how? What can cause the problem?

br
Espen Thingstad

**Comment by AlCalzone** [comment]
This is not an issue in the driver. Please ask in https://github.com/home-assistant/core or the HA forums/discord.
